crates/kotlin-ffi/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 uniffi::setup_scaffolding!();
 
-use alloy::primitives::{Bytes, U256, U64};
+use alloy::primitives::{Bytes as FFIBytes, U256 as FFIU256, U64 as FFIU64};
```

Comment:
Curious: Why do we have to use aliases here?

Reply:
because Bytes are declared both in yttrium and kotlin-ffi and after building bindings swift recognise them as conflicting types